app/src/main/res/values/dimens.xml

```diff
@@ -1,18 +1,19 @@
-<?xml version="1.0" encoding="utf-8"?>
-<resources>
-    <dimen name="dp_50">50dp</dimen>
-    <dimen name="dp_8">8dp</dimen>
-    <dimen name="dp_5">5dp</dimen>
-    <dimen name="dp_10">10dp</dimen>
-    <dimen name="dp_16">16dp</dimen>
-    <dimen name="dp_24">24dp</dimen>
-    <dimen name="dp_22">22dp</dimen>
-    <dimen name="dp_52">52dp</dimen>
-    <dimen name="dp_85">85dp</dimen>
-    <dimen name="dp_122">122dp</dimen>
-    <dimen name="dp_133">133dp</dimen>
-    <dimen name="sp_20">20sp</dimen>
-    <dimen name="sp_22">22sp</dimen>
-    <dimen name="sp_50">50sp</dimen>
-    <dimen name="sp_90">90sp</dimen>
-</resources>
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="dp_50">50dp</dimen>
+    <dimen name="dp_8">8dp</dimen>
+    <dimen name="dp_5">5dp</dimen>
+    <dimen name="dp_10">10dp</dimen>
+    <dimen name="dp_16">16dp</dimen>
+    <dimen name="dp_24">24dp</dimen>
+    <dimen name="dp_22">22dp</dimen>
+    <dimen name="dp_40">40dp</dimen>

```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_40` appears to be unused
 Unused resources make applications larger and slow down builds.